packages/jotai/src/AtomValue.test-d.tsx

```diff
@@ -0,0 +1,21 @@
+import { atom } from 'jotai'
+import type { ReactNode } from 'react'
+import { AtomValue } from './AtomValue'
+
+const asyncAtom = atom(async () => Promise.resolve('string'))
+
+describe('<AtomValue/>', () => {
+  it('type check', () => {
+    ;() => (
+      <AtomValue atom={asyncAtom}>
+        {([value]) => {
+          expectTypeOf(value).toEqualTypeOf<string>()
+          return <></>
+        }}
+      </AtomValue>
+    )
```

Comment:
https://jotai.org/docs/core/use-atom#useatomvalue

In my opinion, This should be just value, not array destructuring

```suggestion
describe('<AtomValue/>', () => {
  it('type check', () => {
    ;() => (
      <AtomValue atom={asyncAtom}>
        {(value) => {
          expectTypeOf(value).toEqualTypeOf<string>()
          return <></>
        }}
      </AtomValue>
    )
```

Reply:
That’s correct. My mistake. Thank you!